### DISCUSSION TITLE
Great's Comparative and Superlative are wrong.

### DISCUSSION THREAD

**Discussion by ByungwooKim-BK** [nlp-compromise/nlp_compromise#1000] [Q&A] [answered]
If I request Great word for getting comparative and Superlative, then I expected Greater and Greatest.
However, actual response are like below. Is there any way to get correct response?

Superlative: 'greattest',
  Comparative: 'greatter',
  Adverb: 'greatly',
  Noun: 'greatness',
  Verb: 'greaten'

**Comment by spencermountain** [comment, marked as answer]
hey @ByungwooKim-BK - this works with `.adjectives().conjugate()` - please read the documentation [here](https://observablehq.com/@spencermountain/compromise-adjectives)
```js
console.log(nlp('great').adjectives().conjugate())
/*[
  {
    Adjective: 'great',
    Comparative: 'greater',
    Superlative: 'greatest',
    Noun: 'greatness',
    Adverb: 'greatly'
  }
]
*/
```
